src/Interactive/HostProcess/InteractiveHostEntryPoint.cs

```diff
@@ -14,7 +13,6 @@ namespace Microsoft.CodeAnalysis.Interactive
 {
     internal static class InteractiveHostEntryPoint
     {
-        [SupportedOSPlatform("windows")]
```

Comment:
😕 I'm not sure what this has to do with "Enums should not have duplicate values"

Reply:
I had to update the roslyn-analyzers version in this PR which introduced [new warnings](https://github.com/dotnet/roslyn/runs/3090507276). This looked to me like the most straightforward way to resolve the warning in this file. I speculate that the new analyzer infers a "default" SupportedOSPlatform based on the target framework of the project.

Reply:
Could you clarify why removing `SupportedOSPlatform` is the solution?

Just to recap, the diagnostic was "src\Interactive\HostProcess\InteractiveHostEntryPoint.cs(23,113): error CA1416: (NETCORE_ENGINEERING_TELEMETRY=Build) This call site is reachable on: 'windows' all versions. 'InteractiveHostEntryPoint.ErrorMode.SEM_NOGPFAULTERRORBOX' is only supported on: 'Windows' 7.0 and later."

Looking at the [doc on CA1416](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1416), it's not clear why removing the SupportedOSPlatform would even fix this diagnostic.
I would have expected that we needed to specify a minimum version of windows instead. Something like `[SupportedOSPlatform("windows10.0.1903")]` (with some proper version where the `SEM_NOGPFAULTERRORBOX` was added).

Reply:
Basically an analyzer thinks it's not OK to use an enum in one of these methods because the enum has a "supported os platform" value that is more restrictive than the caller's value. The enum's value is being inferred as "windows7" from the project while the caller's was explicitly set to "windows". So deleting these attributes causes a consistently restrictive "supported os platform" value to be associated with all the symbols in the project.

Reply:
Got it. So removing the attribute is functionally equivalent to changing it `[SupportedOSPlatform("windows7")]` then (inferred from APIs used in that method)?

Should we revert other parts of commit https://github.com/dotnet/roslyn/pull/48407/commits/c756744877d89d9c083c5e8665b67b5bfccaac99 too then?

Reply:
That seems reasonable to me. Thanks